include/fiction/technology/electrostatic_potential.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef FICTION_ELECTROSTATIC_POTENTIAL_HPP
+#define FICTION_ELECTROSTATIC_POTENTIAL_HPP
+
+//
+// Created by Jan Drewniok on 12.12.22.
+//
+
+#include "fiction/algorithms/simulation_sidb/simulation_parameter.hpp"
+#include "fiction/traits.hpp"
+
+namespace fiction
+{
+/**
+ * The electrostatic potential for a given Euclidean distance is calculated.
+ *
+ * @tparam Dist Euclidean distance, floating-point type for the distance.
+ * @param Potential Floating-point type for the electrostatic potential.
+ * @return electrostatic potential value.
+ */
+template <typename Potential = double, typename Dist = double>
+Potential potential_sidb_pair(const Dist& dist, decltype(simulation_params{}.k) k = simulation_params{}.k,
+                              decltype(simulation_params{}.lambda_tf) lambda_tf = simulation_params{}.lambda_tf)
+{
+    if (dist == static_cast<Dist>(0.0))
```

Comment:
What if `dist` is `1E-19` (or another similarly small value)? Are there tests for it?

Reply:
I see what you mean, but since the smallest nm distance between two adjacent coordinates is never smaller than 2E-10 m, it should be fine, right?

Reply:
But this function does not take SiDB pairs as input. Now that I think about it, it probably should, and then compute the distance itself... :thinking: